CHANGES.rst

```diff
@@ -122,7 +125,6 @@ Specviz2d
 
 - 2D spectrum viewer now has info panel for pixel coordinates and value. [#1608]
 
-Bug Fixes
```

Comment:
Might need another rebase to fix this change log diff weirdness.

Reply:
I rebased for you but didn't go away. I'll push a follow up commit to manually fix this.